docs/FAQ.md

```diff
@@ -8,4 +8,4 @@ Yes, Impostor can be used with both the Android and iOS\* versions of the game.
 ###### \* In order to play on an Impostor server with iOS, you _must_ have a jailbroken device.
 
 ## How can I get started?
-See [Setting up your Server](Running-the-server) for more information on running the server and [Client Setup](https://aeonlucid.com/Impostor/) for helping your friends join in!
\ No newline at end of file
+See [Setting up your Server](Running-the-server) for more information on running the server and [Client Setup](http://impostor.github.io/Impostor/) for helping your friends join in!
```

Comment:
Can we update this to be `https://` please?

Reply:
sure